src/main/java/unrc/dose/UserProfileService.java

```diff
@@ -0,0 +1,17 @@
+package unrc.dose;
+
+import com.google.gson.Gson;
+
+/**
+* Class which calls user profile's methods and transforms they to json.
+*/
+public final class UserProfileService {
+  /**
+  *Invokes the method which creates a user profile and transforms to json.
+  **/
+  public static String userprofile(Integer userId, String displayName, String twitterId) {
+      
+      return UserProfile.createUserProfile(userId, displayName, twitterId).toJson(true);
+
+  }
```

Comment:
The method should be renamed as `create` and `UserProfile` does not seem to have a `toJson` method